internal/backend/dispatcher/eventsworkflow.go

```diff
@@ -24,6 +25,62 @@ func (d *dispatcher) createEventRecord(ctx context.Context, eventID sdktypes.Eve
 	return nil
 }
 
+func (d *dispatcher) resolveEnv(ctx context.Context, env string) (envID sdktypes.EnvID, err error) {
```

Comment:
Not a single comment in ~50 LOC. Please add comments inside this function that explain its flow.

Reply:
fixed